data/fields/license_classes.json

```diff
@@ -0,0 +1,5 @@
+{
+    "key": "license_classes",
+    "type": "semiCombo",
+    "label": "Driving License Classes"
```

Comment:
“Driving license” is understandable but not quite correct in any dialect of English. The source string should be changed to “driver’s license”, per American English. (The British English term would be “driving licence”; the Canadian, Australian, and New Zealand English localizations would have their own translations as well.)

Reply:
Apart from that, “driver’s license classes” is confusing in the context of a preset for driver training. At first, when I tried to translate this string, I thought it meant “classes for obtaining a driver’s license”, but this key actually means “classes of driver’s license”. On the Driver Training Grounds preset, a user might end up leaving this field blank because no classes are taught there. I’d suggest changing this string to “License Classifications” or “Classes of Driver’s License” to avoid confusion.

Reply:
changed the spelling in 7f0b1e4